cli/azd/cmd/provision.go

```diff
@@ -20,9 +20,10 @@ import (
 )
 
 type provisionFlags struct {
```

Comment:
I see a message 'Looking for last deployment state".  I thought it was local, so thought it would be fast.  I think we might want to have a msg like "Retrieving last deployment state from Azure..." something like that.

Reply:
The message is currently:

![image](https://github.com/Azure/azure-dev/assets/24213737/ed084d1b-f981-485a-8dec-3c684ce1d9aa)